test/src/helper/spawn.ts

```diff
@@ -859,7 +859,7 @@ export default class CodeChain {
     public async waitForTermChange(target: number, timeout?: number) {
         const start = Date.now();
         while (true) {
-            const termMetadata = await stake.getTermMetadata(this.sdk);
+            const termMetadata = await stake.getTermMetadata(this.sdk as any);
```

Comment:
It seems to be unrelated. Please remove this change from the PR.

Reply:
Done ✅